docs/metrics/prometheus/README.md

```diff
@@ -104,32 +104,34 @@ Here `prometheus.yml` is the name of configuration file. You can now see MinIO m
 
 MinIO server exposes the following metrics on `/minio/prometheus/metrics` endpoint. All of these can be accessed via Prometheus dashboard. The full list of exposed metrics along with their definition is available in the demo server at https://play.min.io:9000/minio/prometheus/metrics
 
+These are the new set of metrics which will be in effect after `RELEASE.2019-10-02T21-19-38Z`. Some of the key changes in this update are listed below.
+    - Metrics are bound the respective nodes and is not cluster-wide. Each and every node in a cluster will expose its own metrics.
+    - Additional metrics to cover the s3 and internode traffic statistics were added.
+    - Metrics that records the http statistics and latencies are labeled to their respective APIs (putobject,getobject etc).
+    - Disk usage metrics are distributed and labeled to the respective disk paths.
+
+The list of metrics and its definition are as follows. (NOTE: instance here is one MinIO node)
+
+> NOTES:
+    > 1. Instance here is one MinIO node.
+    > 2. `s3 requests` exclude internode requests.
+
+
 - standard go runtime metrics prefixed by `go_`
 - process level metrics prefixed with `process_`
 - prometheus scrap metrics prefixed with `promhttp_`
 
-- `minio_disk_storage_used_bytes` : Total byte count of disk storage used by current MinIO server instance
```

Comment:
What I was thinking was if possible we should suggest them alternatives here, basically talk about existing metrics to a newer one and how to use it.

Reply:
Wont that be too much to feed? Aren't those key changes explained above not enough? The new user might feel confused if they see the alternatives and comparison

Reply:
Anything on this @harshavardhana ?

Reply:
It is for migration, it will make sense to add that IMO

Reply:
Added the migration guide :+1:  PTAL @harshavardhana